tests/test_cases/issue_1279/issue_1279.py

```diff
@@ -0,0 +1,13 @@
+import cocotb
+
+
```

Comment:
```suggestion
# note: this file relies on the fact the simulator runs tests sorted by name
```

Reply:
I instead used the test decorator's `stage`.